cypress/e2e/types/constants.ts

```diff
@@ -40,6 +40,8 @@ export const deleteAction = "Delete";
 export const editAction = "Edit";
 export const description = "Description";
 export const displayName = "Display name";
+export const startDate = "Start date";
+export const endDate = "End date";
```

Comment:
Good idea to add this in a common file as these are words that will probably appear in multiple menus!! :+1: